DocSum/docker/gaudi/docker_compose.yaml

```diff
@@ -71,6 +71,18 @@ services:
       - DOC_BASE_URL=${BACKEND_SERVICE_ENDPOINT}
     ipc: host
     restart: always
+  docsum-xeon-react-ui-server:
+    image: opea/docsum-react-ui:latest
+    container_name: docsum-gaudi-react-ui-server
+    depends_on:
+      - docsum-gaudi-backend-server
+    build:
+      args:
+        - BACKEND_SERVICE_ENDPOINT=${BACKEND_SERVICE_ENDPOINT}
+    ports:
+      - "5174:80"
+    environment:
+      - DOC_BASE_URL=${BACKEND_SERVICE_ENDPOINT}
```

Comment:
The same issue, you didn't use the DOC_BASE_URL variable in your code. Why did you assign a value to this variable? The variable name assigned in your .env file is DOC_SUM_URL.

Reply:
removed environemnt variables